core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -88,6 +93,8 @@ public class LocalCacheManager implements CacheManager {
   private final Optional<ExecutorService> mInitService;
   /** Executor service for execute the async cache tasks. */
   private final Optional<ExecutorService> mAsyncCacheExecutor;
+  /** Executor service for execute the cache ttl check tasks. */
+  private final Optional<ScheduledExecutorService> mTtlEnforcerExecutor;
```

Comment:
Have you tested the new feature in the `LocalCacheManager`? I noticed that there's a corresponding unit test file `LocalCacheManagerTest.java`. Do you want to add some tests there?